opencog/atomspace/AtomTable.cc

```diff
@@ -139,15 +139,15 @@ void AtomTable::clear()
     clear_all_atoms();
 }
 
-Handle AtomTable::getHandle(Type t, const std::string& n) const
+Handle AtomTable::getHandle(Type t, const std::string&& n) const
```

Comment:
As `n` is moved below `std::string&` can be replaced by plain `std::string` safely. Same for `getHandle(Type, HandleSeq)

Reply:
?? There's no more `string&`; I'm changing everything to `string&&`.